rfc-192-use-codeowners-in-all-repos.md

```diff
@@ -0,0 +1,77 @@
+# Use `CODEOWNERS` in all repositories
+
+## Summary
+
+We should configure our repositories with
+[GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+or [GitLab](https://docs.gitlab.com/ee/user/project/code_owners.html)
+`CODEOWNERS` files, to ensure that issues and PRs receive a timely response, and
+that our code is well maintained.
+
+## Problem
+
+`dxw` has a large number of repositories to maintain. At the time of writing,
+GitHub has 260 repositories that are not archived and not forked from elsewhere,
+151 of which are public. It is easy for new PRs and issues to be lost in the
+noise of notifications. For maintainers, it can be difficult to keep up with
+changes in these RFCs, especially for internal code which takes a lower priority
+than client products. For developers, it can be difficult to identify someone to
+review code changes and respond to issues.
+
+## Proposal
+
+For the purposes of this RFC, the term _active repository_ means a repository
+that is:
+
+1. within the `dxw` organisation on GitHub or the equivalent on GitLab,
+1. not a fork of a repository from outside the organisation; and
+1. not archived.
+
+All active repositories MUST have a `CODEOWNERS` file in the root of the
+repository. There SHOULD be more than two members of staff in each file, who
+SHOULD have different roles in the organisation, to avoid all owners of a
+repository being unavailable at once. To save on maintenance time it is
+RECOMMENDED that code owners be a GitHub team, rather than a list of
+individuals.
+
+`CODEOWNERS` files MAY specify different owners for different files, or file
+types, but this should be avoided if it is likely to make it more difficult to
+ensure that contributions are reviewed in a timely manner.
+
+The use of `CODEOWNERS` MUST NOT be used to discourage developers from
+commenting, reviewing or otherwise participating in code maintenance. This is
+particularly important for new or junior members of staff, who should be
+encouraged to participate in code maintenance and review, in a supportive
+environment that
+[assumes good faith](https://en.wikipedia.org/wiki/Wikipedia:Assume_good_faith).
+
+Repositories SHOULD be configured to ensure that code owners review PRs before
```

Comment:
I'm not sure this should be a `SHOULD` without a particular need for the owner(s) to agree to a change, particularly with repos where things like minor Dependabot PRs can be confidently approved by a range of people. A `MAY` is worthwhile to remind people that it's an option though.

Reply:
I might even go as far as removing it. The one time we had it enabled on a repo it was painful. Although its probably okay if its just the tech team as a code owner.